src/Metadata/Tests/Extractor/PropertyMetadataCompatibilityTest.php

```diff
@@ -77,6 +76,7 @@ final class PropertyMetadataCompatibilityTest extends TestCase
         'uriTemplate' => '/sub-resource-get-collection',
         'property' => 'test',
         'hydra' => false,
+        'phpType' => 'string',
```

Comment:
shouldn't this be an array?

Reply:
We'll have only one type at the time: `string`, `string|bool`, etc. So I think it should stay as a string.